### DISCUSSION TITLE
Hat Stack Sanity Check

### DISCUSSION THREAD

**Discussion by jamestalmage** [tagyoureit/nodejs-poolController#730] [Nixie]
Just verifying my planned stack of HAT's is going to work

1. Raspberry PI
2. [Sequent Smart Fan Hat](https://sequentmicrosystems.com/products/smart-fan-for-raspberry-pi?variant=37993169256643)
3. [Sequent Building Automation Hat](https://sequentmicrosystems.com/products/building-automation-8-layer-stackable-hat-v4-for-raspberry-pi?variant=44022451306748)
4. [Sequent 8 Relay Hat](https://sequentmicrosystems.com/products/8-relays-stackable-card-for-raspberry-pi?variant=37993168732355)
5. [Atlas Scientific Whitebox T3](https://atlas-scientific.com/electrical-isolation/whitebox-t3/)
    - EZO PH
    - EZO ORP
    - EZO RTD
6. [Sequent Super Watchdog Hat](https://sequentmicrosystems.com/products/super-watchdog-with-battery-backup-for-raspberry-pi?variant=37993168928963)

My primary questions are about the Super Watchdog, and the Fan Hat. Is the watchdog function implemented? Is fan control (or is the fan control just separate software that is independent of njsPC)?

Other than that, does it seem like a reasonable stack?

**Comment by rstrouse** [comment]
The latest version of the Smart Fan hat has been integrated into REM.  I do not have the Super Watchdog so I cannot comment on that.  @tagyoureit has one of these.

The other hats are fine additions to the stack.  One note though, the Sequent 4 relay you referenced is actually a Sequent 8 relay which most folks are using these days.

**Reply by jamestalmage** [reply]
Yes. I have the 8 relay in my cart, just got confused.

Thanks, as always. Hoping I can repay all the help I'm receiving with some beneficial PR's once I have an actual system to play with.
